ci/dra.sh

```diff
@@ -0,0 +1,22 @@
+#!/bin/bash -ie
+#Note - ensure that the -e flag is set to properly set the $? status if any command fails
+
+# Since we are using the system jruby, we need to make sure our jvm process
+# uses at least 1g of memory, If we don't do this we can get OOM issues when
+# installing gems. See https://github.com/elastic/logstash/issues/5179
+export JRUBY_OPTS="-J-Xmx1g"
+
+# install rake's gme
```

Comment:
unnecessary comment:

```suggestion
```

JRuby already comes with rake, do we really need to install it?

```
/tmp/jruby-9.3.7.0
❯ find . -name "*rake*"
./bin/rake.bat
./bin/rake
./lib/ruby/stdlib/rake
./lib/ruby/stdlib/rake/ant/tasks/raketasks.rb
./lib/ruby/stdlib/rake/ant/rake.rb
./lib/ruby/stdlib/rubygems/ext/rake_builder.rb
./lib/ruby/stdlib/bundler/vendor/thor/lib/thor/rake_compat.rb
./lib/ruby/gems/shared/specifications/default/rake-ant-1.0.4.gemspec
./lib/ruby/gems/shared/specifications/rake-12.3.3.gemspec
./lib/ruby/gems/shared/cache/rake-12.3.3.gem
./lib/ruby/gems/shared/gems/rake-12.3.3
./lib/ruby/gems/shared/gems/rake-12.3.3/bin/rake
./lib/ruby/gems/shared/gems/rake-12.3.3/lib/rake
./lib/ruby/gems/shared/gems/rake-12.3.3/lib/rake/rake_module.rb
./lib/ruby/gems/shared/gems/rake-12.3.3/lib/rake/rake_test_loader.rb
./lib/ruby/gems/shared/gems/rake-12.3.3/lib/rake.rb
./lib/ruby/gems/shared/gems/rake-12.3.3/doc/proto_rake.rdoc
./lib/ruby/gems/shared/gems/rake-12.3.3/doc/rake.1
./lib/ruby/gems/shared/gems/rake-12.3.3/doc/rakefile.rdoc
./lib/ruby/gems/shared/gems/rake-12.3.3/exe/rake
./lib/ruby/gems/shared/gems/rake-12.3.3/rake.gemspec
```

Reply:
So `rake` it's already bundled with JRuby?

Reply:
yeah it comes with a few bundled gems: 

```
/tmp/jruby-9.3.7.0
❯ ls ./lib/ruby/gems/shared/gems
bundler-2.2.29		jar-dependencies-0.4.1	power_assert-1.1.3	rdoc-6.1.2.1		xmlrpc-0.3.0
did_you_mean-1.3.0	minitest-5.11.3		racc-1.5.2-java		rubygems-update-3.2.29
irb-1.0.0		net-telnet-0.1.1	rake-12.3.3		test-unit-3.2.9

```